src/SFA.DAS.AssessorService.Application.Api.External/Startup.cs

```diff
@@ -63,6 +62,53 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             {
                 ApplicationConfiguration = ConfigurationService.GetConfigExternalApi(Configuration["EnvironmentName"], Configuration["ConfigurationStorageConnectionString"], VERSION, SERVICE_NAME).Result;
 
+                services.AddAuthentication(auth =>
+                {
+                    auth.DefaultScheme = JwtBearerDefaults.AuthenticationScheme;
+                })
+                .AddJwtBearer(auth =>
+                {
+                    var validAudiences = new List<string>();
+                    var tenant = string.Empty;
+
+                    if (_useSandbox)
+                    {
+                        validAudiences.AddRange(ApplicationConfiguration.SandboxExternalApiAuthentication.Audiences.Split(","));
+                        tenant = auth.Authority = ApplicationConfiguration.SandboxExternalApiAuthentication.Tenant;
+                    }
+                    else
+                    {
+                        validAudiences.AddRange(ApplicationConfiguration.ExternalApiAuthentication.Audiences.Split(","));
+                        tenant = auth.Authority = ApplicationConfiguration.ExternalApiAuthentication.Tenant;
+                    }
+
+                    auth.Authority = $"https://login.microsoftonline.com/{tenant}";
```

Comment:
And then setting it to a formatted string - so why set it above?

Reply:
I have removed the first auth.Authority assignment